crates/runix/src/types/narinfo.rs

```diff
@@ -0,0 +1,16 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::types::DerivationPath;
+
+/// Narinfo stores information output by `nix path-info --json`
+#[derive(Serialize, Deserialize, Clone)]
+pub struct Narinfo {
+    pub path: DerivationPath,
+    pub valid: bool,
```

Comment:
lets add the default true back in here, at least for a while in support of nix <2.15